### DISCUSSION TITLE
Automatic conversion from traditional to simplified Chinese

### DISCUSSION THREAD

**Discussion by JulianGro** [WeblateOrg/weblate#9042] [Q&A]
Is there any way to use Weblate to automatically generate a simplified Chinese translation from a traditional Chinese translation?

My understanding is that traditional Chinese characters can be mapped to simplified Chinese characters without having to translate the actual text itself.
I would assume that translation systems like DeepL would realize this and just convert the characters, but even then, I don't see any option to machine translate a specific locale from another specific locale.

**Comment by nijel** [comment]
I don't know about the actual differences between these two, so I can't comment on that.

DeepL supports only simplified Chinese, but Google should work fine for translating between the two.
